examples/js/controls/TransformControls.js

```diff
@@ -532,6 +532,29 @@
 
 		}
 
+		reset( stopCurrentTransform ) {
```

Comment:
You have to modify `examples/jsm/controls/TransformControls.js` instead.

Reply:
I did both, but I understand only the jsm one has to be changed?

Reply:
Correct. Sorry, I did not see the changes to `examples/jsm/controls/TransformControls.js `.